src/routes/NewRequest/components/NewRequestPage/useNewRequestPage.js

```diff
@@ -61,6 +60,7 @@ export default function useNewRequestPage() {
       alert('Please select a location by clicking on the map above.'); // eslint-disable-line no-alert
       return;
     }
+    delete requestLocation.lookedUpAddress;
```

Comment:
This works for now, but I personally usually avoid using delete since it is modifying an existing object which can sometimes have unintended consequences. I usually create a new object without that value, and if it is only one needing to be removed, you can use rest spread

Reply:
Fixed.